Tests/OAuth/ResourceOwner/GenericOAuth1ResourceOwnerTest.php

```diff
@@ -51,10 +51,14 @@ public function setUp()
     }
 
     /**
-     * @expectedException \Symfony\Component\OptionsResolver\Exception\InvalidOptionsException
+     * @expectedException \Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException
```

Comment:
you could just expect the ExceptionInterface of the component. It would be simpler